### PR TITLE
[POC] Add translatable input to window object to make it accessible globally

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This POC aims to help modules using core dependencies by binding these dependencies to the window object
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20313)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/theme.js [hunk 1]
```diff
@@ -62,6 +63,13 @@ new NavBar();
 new Header();
 
 $(() => {
+  // Initialize the prestashop global object
+  window.prestashop = {
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/theme.js:67]: `prestashop` is already initialized watch out.

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/js/theme.js:67]: BO doesn't have a prestashop object

**Review by matks — APPROVED** [review]
Looks like a nice solution for now 👍

**Comment by NeOMakinG** [comment]
Closing as it has been implemented after an ADR

**Pull request closed without merging**